### PR TITLE
chore: clone last 5 commits

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 language: node_js
 node_js:
   - '6'
+git:
+  depth: 5
```
Comment (FagnerMartinsBrack) [inline .travis.yml]: What's the difference between the default depth of `50` and using 5 instead?

Comment (DanielRuf) [reply .travis.yml]: `5` is often faster

Comment (DanielRuf) [reply .travis.yml]: Cloning 5 vs 50 commits makes a big difference in commit changes / diffs.

Comment (FagnerMartinsBrack) [reply .travis.yml]: How fast are we talking about compared to the whole build? Milliseconds or seconds?

Comment (DanielRuf) [reply .travis.yml]: Just milliseconds so probably not worth it.

**Pull request closed without merging**